lib/errors.js

```diff
@@ -38,9 +38,23 @@ util.inherits(error.BaseError, Error);
  * @constructor
  */
 error.ValidationError = function(message, errors) {
+  var self = this;
+
   error.BaseError.apply(this, arguments);
   this.name = 'SequelizeValidationError';
   this.errors = errors || [];
+
+  if (this.errors.length > 0 && this.errors[0].message) {
+    this.message = "";
+
+    this.errors.forEach(function(_err, i) {
```

Comment:
`this.errors` is not string. I have to join `this.errors[0].message`, `this.errors[1].message`, `this.errors[2].message`..., so `forEach` is required.